content/en/guides/get-started/installation.md

```diff
@@ -8,29 +8,71 @@ csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/
 
 ## Prerequisites
 
-Here, you will find information on setting up and running a Nuxt.js project in 4 steps.
+- [node](https://nodejs.org/en/download/) - at least v8.9.0 _We recommend you have the latest version installed._
+- Text editor: Use whatever you like, but we recommend [VSCode](https://code.visualstudio.com/).
```

Comment:
Webstorm also has a really good support for nuxt.js. We can maybe mention both? There is no recommendation point

Reply:
Agree, will add a link to WebStorm